spec/shared_examples/cli_parsing.rb

```diff
@@ -17,4 +17,12 @@
     expect(stderr.string).to be_empty
     expect(first_code).to be 0
   end
+
+  it 'raise the verbose flag' do
+    args = ['--verbose', action].compact
+    _,parsed,_ = Puppetserver::Ca::Cli.parse_general_inputs(args)
+    expect(parsed['verbose']).to be true
+    logger = Puppetserver::Ca::Logger.new(:debug, stdout, stderr)
+    expect(logger.level).to be Puppetserver::Ca::Logger::LEVELS[:debug]
```

Comment:
hmm, since you're creating a new `logger` here, I'm not sure that this is actually testing that the CLI inputs are correctly passed to the Logger class. Does the test still pass if you remove lines 22-24?

Reply:
It does. I just wanted to make sure that the level number is correct for the sake of it.

Reply:
no yeah, I think that's fine, but I think you need something else to confirm that passing the `--verbose` flag ends up creating a Logger at the debug level. Does that make sense? Happy to chat more about it tomorrow.

Reply:
Yep that crossed my mind but I did not know how to express that because the logger that will be created will be a local variable to the function `run`. And I wasn't sure how I can check that local variable.

Reply:
yeah, I wasn't sure off the top of my head exactly how to test that haha, but happy to pair on it tomorrow if you want!